src/zope/app/apidoc/codemodule/browser/static_menu.pt

```diff
@@ -1,48 +1,39 @@
 <html metal:use-macro="context/@@apidoc_macros/static_menu"
-    i18n:domain="zope">
+      i18n:domain="zope">
 <body>
 
   <div metal:fill-slot="menu" class="small">
 
     <div>
-      <span i18n:translate="">Class Finder:</span>
-      <span><i i18n:translate="">(Enter partial Python path)</i></span>
-	</div>
-    <form action="#" method="post" name="searchform" >
-	  <br />
+      <span i18n:translate="">Class Finder:</span><br />
+      <em i18n:translate="">(Enter partial Python path)</em>
+    </div>
+    <form action="" method="post" name="searchform"
+          onsubmit="simplegetSearchResult(document.searchform.path.value);
+                    return false">
       <input type="text" name="path"
-             style="font-size: 80%; width=95%" />
-
-	  <br />
-      <input type="button" name="Find" value="Find"
-	     onClick="
-               javascript:simplegetSearchResult(document.searchform.path.value)"
-             i18n:attributes="value find-button" style="font-size: 80%"/>
-
-<!--
-      <input type="submit" name="SUBMIT" value="Find"
-             i18n:attributes="value find-button" style="font-size: 80%"/>
-  -->
-  </form>
+             style="font-size: 80%; width: 95%" />
+      <input type="submit" name="Find" value="Find"
+             i18n:attributes="value find-button" style="font-size: 80%" />
+    </form>
 
     <p style="font-size: 120%">
       <a href="./index.html" target="main" i18n:translate="">Browse
         Zope Source</a>
     </p>
 
-	<div>
+    <div>
       <span i18n:translate="">Search results:</span><br /><br />
     </div>
 
-    <div tal:define="pyclasses view/findAllClasses"
-         tal:condition="pyclasses" id="resultlist">
+    <div tal:define="classes view/findAllClasses"
```

Comment:
I don't understand why it was necessary to change this variable name?

Reply:
It wasn't